Code/Tools/HeaderCheck/HeaderCheck.cpp

```diff
@@ -406,7 +406,8 @@ class ezHeaderCheckApp : public ezApplication
       ezStringBuilder currentFileLower = sCurrentFile.GetFileNameAndExtension();
       currentFileLower.ToLower();
 
-      bool ignore = m_IgnoreTarget.m_byName.Contains(includeFileLower) || m_IgnoreSource.m_byName.Contains(currentFileLower);
+      // ignore includes ending in "_Platform.h", they redirect to platform specific EZ headers
+      const bool ignore = m_IgnoreTarget.m_byName.Contains(includeFileLower) || m_IgnoreSource.m_byName.Contains(currentFileLower) || includeFileLower.EndsWith_NoCase("_Platform.h");
```

Comment:
Note this addition, to automatically ignore includes of the form

```cpp
#include <Stuff_Platform.h>
```

Otherwise every platform specific include is assumed to be a 'system include' and reported as an error.